src/reuse/download.py

```diff
@@ -31,6 +34,8 @@
     "https://raw.githubusercontent.com/spdx/license-list-data/master/text/"
 )
 
+REF_RE = re.compile("LicenseRef-.+")
```

Comment:
This will restrict the charset to ASCII only which might be too restrictive depending on the country where the tool is run.

Reply:
Agreed. Then again SPDX dictates ASCII in the way they refer to the RFC5234 standard https://datatracker.ietf.org/doc/html/rfc5234#appendix-B.1 It is repeated throughout the specification https://github.com/search?q=org%3Aspdx+ascii&type=code So I think it makes sense to check this upfront, so that further processing will comply with SPDX.

Reply:
Oh, I missed that part. Then yes, the check is a good as suggested.